controllers/admin/AdminImportController.php

```diff
@@ -3127,11 +3141,15 @@ protected function addressImportOne($info, $force_ids, $validateOnly = false)
                 $customer_list = Customer::getCustomersByEmail($address->customer_email);
 
                 if (count($customer_list) == 0) {
-                    $this->errors[] = sprintf(
-                        Tools::displayError('%1$s does not exist in database %2$s (ID: %3$s), and therefore cannot be '.($validateOnly?'validated':'saved')),
-                        Db::getInstance()->getMsgError(),
-                        $address->customer_email,
-                        (isset($info['id']) && !empty($info['id']))? $info['id'] : 'null'
+                    $this->errors[] = $this->trans(
+                        '%1$s does not exist in database %2$s (ID: %3$s), and therefore cannot be %4$s',
```

Comment:
This should also be split into two strings:
* `%1$s does not exist in database %2$s (ID: %3$s), and therefore cannot be validated`
* `%1$s does not exist in database %2$s (ID: %3$s), and therefore cannot be saved`

Also, it seems that:
* %1$s is a DB error message: `Db::getInstance()->getMsgError()`
* %2$s is the customer's email: `$address->customer_email`

Does the generated result even make sense?